x-pack/plugins/code/server/indexer/lsp_indexer.ts

```diff
@@ -193,17 +193,17 @@ export class LspIndexer extends AbstractIndexer {
     const lstat = util.promisify(fs.lstat);
     const stat = await lstat(localFilePath);
 
+    if (stat.size > TEXT_FILE_LIMIT) {
```

Comment:
actual change here

Reply:
we might in the future record how many file has been skipped index

Reply:
add a new key to the returned `stats` should be good. let's polish the stats later.